### ISSUE TITLE
Messages that do not arrive

### ISSUE THREAD

**Issue by Gh0stz0x** [deltachat/deltachat-android#2369]
- Android version: Android 11
- Device: WIKO W-V745-EEA
- Delta Chat version: 1.32.0
- Expected behavior: When I send geo-located photos they must arrive.
- Actual behavior: When I send geo-located photos, they don't arrive
- Steps to reproduce the problem: Enable geolocation and send some photos. If you click on the photo sending area on the recipient's map, a Trash group opens for you.
- Screenshots: 
![deltachat-2022-08-21-121522](https://user-images.githubusercontent.com/74795149/185787025-7266ba3f-16c3-4ca7-bde4-afa4e7c5f297.jpg)

- Logs:

**This log may contain sensitive information. If you want to post it publicly you may examine and edit it beforehand.**

device=WIKO W-V745-EEA (W-V745-EEA)
android=11 (1653633227, RP1A.200720.011 release-keys)
sdk=30
memory=18M (6.33% free, 256M max)
memoryClass=128, low-mem device
host=ubuntu
applicationId=chat.delta
app=Delta Chat 1.32.0-gplay
installer=com.google.android.packageinstaller
ignoreBatteryOptimizations=true
notifications=true
reliableService=true
lang=it_IT
rtl=false

arch=32
bcc_self=0
blobdir=/data/user/0/chat.delta/files/accounts/dd91d624-a7f3-4cef-9c46-4f41c27779c3/dc.db-blobs
bot=0
configured_mvbox_folder=DeltaChat
configured_sentbox_folder=Sent
database_dir=/data/user/0/chat.delta/files/accounts/dd91d624-a7f3-4cef-9c46-4f41c27779c3/dc.db
database_encrypted=false
database_version=91
delete_device_after=0
delete_server_after=0
deltachat_core_version=v1.90.0
display_name=****
download_limit=0
e2ee_enabled=1
entered_account_settings=****@****.** imap:unset:***:mail.infomaniak.com:0:cert_automatic:AUTH_NORMAL smtp:unset:0:mail.infomaniak.com:0:cert_automatic:AUTH_NORMAL
fetch_existing_msgs=1
fetched_existing_msgs=true
fingerprint=*************
folders_configured=3
imap_server_id=Some({"name": "Dovecot"})
is_configured=1
journal_mode=wal
key_gen_type=0
last_housekeeping=1661021464
level=awesome
mdns_enabled=1
media_quality=0
messages_in_contact_requests=0
mvbox_move=0
num_cpus=8
number_of_chat_messages=301
number_of_chats=11
number_of_contacts=8
only_fetch_mvbox=0
private_key_count=1
public_key_count=6
quota_exceeding=0
scan_all_folders_debounce_secs=60
secondary_addrs=
selfavatar=/data/user/0/chat.delta/files/accounts/dd91d624-a7f3-4cef-9c46-4f41c27779c3/dc.db-blobs/selfavatar4068024968740531537.jpg
send_sync_msgs=0
sentbox_watch=0
show_emails=2
socks5_enabled=0
sqlite_version=3.37.2
uptime=38h 6m 9s
used_account_settings=****@***.*** imap:****@****.**:***:mail.infomaniak.com:143:cert_strict:AUTH_NORMAL smtp:****@****.**:***:mail.infomaniak.com:587:cert_strict:AUTH_NORMAL
webrtc_instance=https://meet.jit.si/$ROOM

--------- beginning of main
08-21 12:17:05.836  6696  6696 I ConversationFragment: ⏰ getChatMsgs(27): 25ms
08-21 12:18:05.813  6696  6696 I ConversationFragment: ⏰ getChatMsgs(27): 3ms
08-21 12:19:05.813  6696  6696 I ConversationFragment: ⏰ getChatMsgs(27): 2ms
08-21 12:20:05.813  6696  6696 I ConversationFragment: ⏰ getChatMsgs(27): 2ms
08-21 12:21:05.817  6696  6696 I ConversationFragment: ⏰ getChatMsgs(27): 6ms
08-21 12:21:22.561  6696  6696 I PassphraseRequiredActionBarActivity: onDestroy()
08-21 12:21:22.640  6696  6696 D View    : [Warning] assignParent to null: this = DecorView@3afdcf4[ConversationActivity]
08-21 12:21:22.649  6696  6696 I InputTransport: Destroy ARC handle: 0x82cbef00
08-21 12:21:22.650  6696  6696 I PassphraseRequiredActionBarActivity: onDestroy()
08-21 12:21:22.668  6696  6696 D View    : [Warning] assignParent to null: this = DecorView@e0293a6[ConversationListRelayingActivity]
08-21 12:21:22.695  6696  6696 I InputTransport: Destroy ARC handle: 0x84158450
08-21 12:21:22.720  6696  6696 W PassphraseRequiredActionBarActivity: onCreate(null)
08-21 12:21:22.835  6696  6696 I DeltaChat: ++++++++++++++++++ first ForegroundDetector.onActivityStarted() ++++++++++++++++++
08-21 12:21:22.836  6696  6762 I DeltaChat: src/context.rs:218: starting IO
08-21 12:21:22.838  6696 22990 I DeltaChat: calling maybeNetwork()
08-21 12:21:22.840  6696 22990 I DeltaChat: maybeNetwork() returned
08-21 12:21:22.840  6696  6762 I DeltaChat: src/scheduler.rs:379: smtp fake idle - interrupted
08-21 12:21:22.841  6696  6762 I DeltaChat: src/imap/idle.rs:83: Idle wait was interrupted
08-21 12:21:22.843  6696  6762 I DeltaChat: src/smtp.rs:634: Sending MDNs
08-21 12:21:22.844  6696  6762 I DeltaChat: src/scheduler.rs:353: smtp fake idle - started
08-21 12:21:22.844  6696  6762 I DeltaChat: src/scheduler.rs:375: smtp has no messages to retry, waiting for interrupt
08-21 12:21:22.862  6696 22694 I ConversationListFragment: executing debounced chatlist loading
08-21 12:21:22.871  6696  6696 I PassphraseRequiredActionBarActivity: onResume()
08-21 12:21:22.874  6696  6696 V PhoneWindow: DecorView setVisiblity: visibility = 4, Parent = null, this = DecorView@7b39258[]
08-21 12:21:22.876  6696  6696 D ViewRootImpl[RoutingActivity]: hardware acceleration = true , fakeHwAccelerated = false, sRendererDisabled = false, forceHwAccelerated = false, sSystemRendererDisabled = false
08-21 12:21:22.889  6696  6696 I InputTransport: Create ARC handle: 0x971e8d30
08-21 12:21:22.890  6696  6696 V PhoneWindow: DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@5f535ed, this = DecorView@7b39258[RoutingActivity]
08-21 12:21:23.028  6696  6762 I DeltaChat: src/job.rs:350: loading job
08-21 12:21:23.148  6696  6762 I DeltaChat: src/imap.rs:904: 0 mails read from "INBOX".
08-21 12:21:23.155  6696  6762 I DeltaChat: src/imap/scan_folders.rs:28: Starting full [...]

**Comment by GitKKg** [comment]
Agree, got same issue,same android 11 OS ,same version 1.32.0, even if keep Delta Chat foregrounded all the time, it will not update msg, but if close or minimize it ,then foreground  it again, then it will update msg at once, disconnect the network and reconnect network again will take same effect.
Same version of PC app just works normal

I made some analysis on log, seems this section matters:
**08-22 18:56:50**.777 31993 32033 I DeltaChat: src/imap/idle.rs:57: **INBOX: Idle entering wait-on-remote state**
**08-22 18:57:09**.559 31993 31993 I PassphraseRequiredActionBarActivity: onPause()
08-22 **18:57:10**.336 31993 31993 I DeltaChat: ++++++++++++++++++ **last ForegroundDetector.onActivityStopped**() ++++++++++++++++++
08-22 18:57:10.339 31993 31993 I PassphraseRequiredActionBarActivity: onDestroy()
08-22 18:59:49.589 31993 32477 I DeltaChat: ++++++++++++++++++ FetchWorker.doWork() started ++++++++++++++++++
08-22 18:59:49.589 31993 32033 I DeltaChat: src/context.rs:218: starting IO
08-22 18:59:50.204 31993 31993 W PassphraseRequiredActionBarActivity: onCreate(null)
08-22 18:59:50.229 31993 31993 I DeltaChat: ++++++++++++++++++ first ForegroundDetector.onActivityStarted() ++++++++++++++++++
08-22 18:59:50.229 31993 32033 I DeltaChat: src/context.rs:218: starting IO
08-22 18:59:50.229 31993 32486 I DeltaChat: calling maybeNetwork()
08-22 18:59:50.229 31993 32486 I DeltaChat: maybeNetwork() returned
08-22 18:59:50.230 31993 32033 I DeltaChat: src/imap/idle.rs:83: **Idle wait was interrupted**
08-22 18:59:50.230 31993 32033 I DeltaChat: src/imap/idle.rs:83: Idle wait was interrupted

seems Idle is never interrupted when you keep DeltaChat foregrounded from before Msg is actually sent from another client side.
same to keep DeltaChat backgrounded or minimized from before Msg is actually sent from another client side, but log is tricky to be got for me.

**Comment by gerryfrancis** [comment]
@adbenitez confirmed that no EXIF GPS data of JPEG images are processed for geolocation, so this seems to be a general location streaming issue.

**Comment by adbenitez** [comment]
@link2xt isn't this a core issue?
